parallel_gripper_controller/include/parallel_gripper_controller/parallel_gripper_action_controller_impl.hpp

```diff
@@ -317,7 +348,12 @@ controller_interface::CallbackReturn GripperActionController::on_activate(
   }
 
   // Command - non RT version
-  command_struct_.position_cmd_ = joint_position_state_interface_->get().get_value();
+  const auto position_op = joint_position_state_interface_->get().get_optional();
+  if (!position_op.has_value())
+  {
+    RCLCPP_DEBUG(get_node()->get_logger(), "Unable to retrieve data of joint position");
+  }
+  command_struct_.position_cmd_ = position_op.value();
```

Comment:
what should happen here if has_value is false?

Reply:
Same here?

Reply:
```suggestion
  else
  {
    command_struct_.position_cmd_ = position_op.value();
  }
```